daemon/README.md

```diff
@@ -0,0 +1,176 @@
+Daemon container
+================
+
+This Dockerfile may be used to bootstrap a Ceph cluster with all the Ceph daemons running.
+
+
+Usage
+-----
+
+You can use this container to bootstrap any Ceph daemon.
+
+* `CEPH_DAEMON` is the name the daemon you want to deploy (DEFAULT: none). Available values: `MON`, `OSD_DIRECTORY`, `OSD_CEPH_DISK`, `MDS`, `RGW`.
+* `CLUSTER` is the name of the cluster (DEFAULT: ceph)
+* `HOSTNAME` is the hostname of the machine  (DEFAULT: $(hostname))
+
+
+Deploy a monitor
+----------------
+
+Run:
+
+```
+$ sudo docker run -d --net=host \
+-v /etc/ceph:/etc/ceph \
+-v /var/lib/ceph/:/var/lib/ceph/ \
+-e CEPH_DAEMON=MON \
+-e MON_IP=192.168.0.20 \
+-e CEPH_PUBLIC_NETWORK=192.168.0.0/24 \
+ceph/daemon
+```
+
+List of available options:
+
+* `MON_IP`: IP address of the host running Docker
+* `MON_NAME`: name of the monitor (default to hostname)
+* `CEPH_PUBLIC_NETWORK`: CIDR of the host running Docker, it should be in the same network as the `MON_IP`
+* `CEPH_CLUSTER_NETWORK`: CIDR of a secondary interface of the host running Docker. Used for the OSD replication traffic
+
+
+Deploy an OSD
+-------------
+
+There are two available options:
+
+* use `OSD_CEPH_DISK` where you only specify a block device
+* use `OSD_DIRECTORY` where you specify an OSD mount point to your container
+
+
+### Ceph disk ###
```

Comment:
Yes we should clarify that preparation steps are needed before running the container and exposing the OSD directory.